client/src/components/ErrorMessageComponent/ErrorMessage.js

```diff
@@ -23,7 +25,8 @@ export class ErrorMessageComponent extends Component {
       showAuthSpinner: false,
       showAuthStatus: 0,
       showAuthResponse: false,
-      stackErrorMessage: ""
+      stackErrorMessage: "",
+      appAdtChinaUrl: false
```

Comment:
A better name might be `showAdtChinaWarningMessage` or similar?

Reply:
Ready